src/main/java/com/cgdpd/library/service/BookService.java

```diff
@@ -24,4 +28,10 @@ public BookDTO createBook(CreateBookRequestDTO requestDTO) {
         var createdBook = bookRepository.save(bookEntity);
         return bookMapper.mapToBookDTO(createdBook);
     }
+
+    public Page<BookDTO> getBooks(int page, int size) {
```

Comment:
Create object to have the sort criteria, page index, page size, and sort params as well